packages/common/test/deserializeSchema.test.ts

```diff
@@ -31,3 +31,32 @@ test('Deserialize V2 schema from file (icecream)', async () => {
     expect(parsedModule.value[contractName].init).toBeDefined();
     expect(parsedModule.value[contractName].receive).toBeDefined();
 });
+
+test('Deserialize V1 versioned schema from file (cis1-wccd)', async () => {
+    const rawModule = Buffer.from(
+        fs.readFileSync('./test/resources/cis1-wccd-schema-v1-versioned.bin')
+    );
+    const parsedModule = deserialModuleFromBuffer(rawModule);
+
+    const contractName = 'CIS1-wCCD';
+
+    if (parsedModule.v != SchemaVersion.V1) {
+        throw new Error('Unexpected schema version');
+    }
+
+    expect(parsedModule.value[contractName].state).toBeDefined();
+    expect(parsedModule.value[contractName].init).toBeDefined();
+    expect(parsedModule.value[contractName].receive).toBeDefined();
+});
+
+test('Deserialize V2 versioned schema from file (cis2-wccd)', async () => {
+    const rawModule = Buffer.from(
+        fs.readFileSync('./test/resources/cis2-wccd-schema-v2-versioned.bin')
+    );
+    const parsedModule = deserialModuleFromBuffer(rawModule);
+
```

Comment:
Please add a check to ensure that this is a v2 schema. (Similar to the v1 test above)